templates/ingress.yaml

```diff
@@ -20,6 +20,12 @@ metadata:
     appgw.ingress.kubernetes.io/ssl-redirect: "true"
 {{- else if eq .Values.ingress.class "nginx" }}
     nginx.ingress.kubernetes.io/ssl-redirect: "true"
+{{- else if eq .Values.ingress.class "alb" }}
+    # https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.2/guide/ingress/annotations/
+    alb.ingress.kubernetes.io/scheme: "internet-facing"
+    alb.ingress.kubernetes.io/ssl-redirect: "443"
+    alb.ingress.kubernetes.io/target-type: "ip"
+    alb.ingress.kubernetes.io/healthcheck-path: /up
```

Comment:
We should use ssl policy ELBSecurityPolicy-TLS-1-2-2017-01 to disable TLS 1 and 1.1 (we use this in the current terraform)

Reply:
Fixed and verified.